AppCenter/AppCenter/MSService.h

```diff
@@ -11,24 +11,12 @@
  */
 @protocol MSService <NSObject>
 
-/**
- * Enable or disable this service.
- * The state is persisted in the device's storage across application launches.
- *
- * @param isEnabled Whether this service is enabled or not.
- *
- * @see isEnabled
- */
-+ (void)setEnabled:(BOOL)isEnabled;
-
 /**
  * Indicates whether this service is enabled.
  *
- * @return `YES` if this service is enabled, `NO` if it is not.
- *
- * @see setEnabled:
+ * The state is persisted in the device's storage across application launches.
  */
-+ (BOOL)isEnabled;
+@property(class, nonatomic, assign, setter=setEnabled:) BOOL isEnabled;
```

Comment:
Nit: by looking at [Apple naming conventions](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingIvarsAndTypes.html) this property should be named `enabled` with specified getter. Something like: 
```suggestion
@property(class, nonatomic, assign, getter=isEnabled:) BOOL enabled;
```
Should apply to other similar `isAdjective` properties in this PR.

Reply:
Good call. I believe this is the only `isAdjective` change in this PR.

Reply:
Sorry, the only _writable_ one.

Reply:
Fixed by a9987d9625a2ea8ab1c607cd78846c692ac046e3.

Reply:
Sounds good thanks.